app/models/concerns/twitch_user.rb

```diff
@@ -12,7 +12,7 @@ def follows
     end
 
     def twitch_sync!
-      body = Twitch::User.get(name)
+      body = Twitch::User.get(self)
 
       basic_info = JSON.parse(body)
 
```

Comment:
Can you also add `twitch_display_name` to the attributes we update in this section? I just realized we don't do it here but we do in `SessionsController#create`.